src/test/java/echoserver/EchoServerTest.java

```diff
@@ -2,50 +2,108 @@
 
 import org.junit.jupiter.api.Test;
 
-import java.io.*;
+import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 
 import static org.junit.jupiter.api.Assertions.*;
 
 class EchoServerTest {
     @Test
-    void itEchosText() throws IOException {
-        EchoServer echoServer = new EchoServer();
-        ArrayList<String> input = new ArrayList<String>(Arrays.asList("This", "Should", "be", "echoed"));
-        TestConnection connection = new TestConnection(input);
-        echoServer.echo(connection);
+    void itConnectsToClientsInOrderReceived() throws IOException {
+        FakeEchoClient echoClient = new FakeEchoClient();
+        ClientSocketConnection first_client = new ClientSocketConnection();
+        ClientSocketConnection second_client = new ClientSocketConnection();
+        ClientSocketConnection third_client = new ClientSocketConnection();
```

Comment:
Looks like there are varying styles here. `snake_case` and `camelCase`

Reply:
Oops